### DISCUSSION TITLE
Clipboard ui doesnt pop up using keybind

### DISCUSSION THREAD

**Discussion by dastagirS** [end-4/dots-hyprland#828] [General]
fuzzel from terminal successfully runs tho clipboard, emoji using Super +v and Super+, doesnt work. it used to work. How do i debug this ive only changed the foot conf to use ctrl+shift+c and ctrl+shift+v for copy and paste and nothing else.

**Comment by SamuelHeinrichs** [comment]
See if this is your problem, solved for me: [https://github.com/end-4/dots-hyprland/issues/805](url)

**Reply by dastagirS** [reply]
thanks that solved the problem.
